src/context/Theme.tsx

```diff
@@ -105,6 +105,10 @@ const themeSettings = createTheme({
                         right: 0,
                         top: 0,
                         width: '100%',
+                        zIndex: 5,
+                    },
+                    '& 	.MuiChip-deleteIcon': {
+                        zIndex: 10,
```

Comment:
The custom styling above is what prevents the delete chip from working. This just makes sure the zIndex is higher.